packages/tectonic-explorer/js/stores/simulation-store.ts

```diff
@@ -769,7 +772,7 @@ export class SimulationStore {
       version: 1,
       answerType: "interactive_state",
       dataSampleColumns: config.dataSampleColumns,
-      dataSamples: toJS(this.dataSamples) // copy MobX observable array to plain JS array
+      dataSamples: JSON.parse(JSON.stringify(this.dataSamples)) // copy MobX observable array to plain JS array
```

Comment:
Just curious why you changed it to this.  Does `toJS` not do deep copies?

Reply:
I have committed that accidentally. It was part of my debugging and I was testing if it wasn't a problem. `toJS` is enough, I'll update it in the future comments. Thanks.